packages/zowe-explorer/CHANGELOG.md

```diff
@@ -4,8 +4,16 @@ All notable changes to the "vscode-extension-for-zowe" extension will be documen
 
 ## TBD Release
 
+### Bug fixes
+
+- Added Time Started, Time Ended, and Time Submitted job properties to the Jobs table view. [#3055](https://github.com/zowe/zowe-explorer-vscode/issues/3055)
+- Implemented copy/paste functionality of data sets within and across LPARs. [#3012](https://github.com/zowe/zowe-explorer-vscode/issues/3012)
+- Implemented drag and drop functionality of data sets within and across LPARs. [#3413](https://github.com/zowe/zowe-explorer-vscode/pull/3413)
+- Fixed an issue where copy-and-pasting an empty PDS within the same LPAR would result in an error. [#3541](https://github.com/zowe/zowe-explorer-vscode/issues/3541)
+
 ### New features and enhancements
 
+- Added functionality for pagination in job spool files and added the command's key binding [#1787](https://github.com/zowe/zowe-explorer-vscode/issues/1787)
```

Comment:
we should share the keybinding for this action in the CHANGELOG entry for awareness

Reply:
also make sure it doesn't break use in different OSs as we have done and had to fix in the past, thanks